runtime/src/lib.rs

```diff
@@ -46,8 +66,12 @@ pub use sp_runtime::{Perbill, Permill};
 pub type BlockNumber = u32;
 
 /// Alias to 512-bit hash when used in the context of a transaction signature on the chain.
+#[cfg(not(feature = "standalone"))]
 pub type Signature = MultiSignature;
 
+#[cfg(feature = "standalone")]
+pub type Signature = sr25519::Signature;
```

Comment:
Hi @olanod, can I switch to `MultiSignature` for node aswell?

Reply:
Sure, just check if any pallet complains or something. I think I left it that way to avoid some old code from complaining.